test/useForm.test.tsx

```diff
@@ -84,6 +84,22 @@ describe('useForm', () => {
     expect(result.current.isEmpty).toEqual(false)
   })
 
+  it('should allow false-y values as default values', () => {
+    type SomeType = {
+      name: string
+      isPresent: boolean
+    }
+
+    const { result } = render<SomeType>({
+      name: field({ default: '', rules: [] }), // empty strings are falsey in JS, e.g. "" || "foo" => "foo"
+      isPresent: field<boolean>({ default: false, rules: [] })
```

Comment:
Passing `<boolean>` here was necessary as TS was trying to infer the generic as `false`.